torch/_inductor/ir.py

```diff
@@ -3687,9 +3725,11 @@ def create(cls, inputs, dim):
 
         output_stride = FlexibleLayout.contiguous_strides(new_size)
         # If any of the inputs is in CL format, use CL format for the output
+        any_input_is_storage_and_layout = False
```

Comment:
`any_input_is_storage_and_layout = any(is_storage_and_layout(x) for x in inputs)` looks clearer?